src/gf.c

```diff
@@ -388,14 +388,15 @@ jl_code_info_t *jl_type_infer(jl_method_instance_t *mi, size_t world, int force)
     }
     JL_CATCH {
         jl_value_t *e = jl_current_exception();
+        jl_printf((JL_STREAM*)STDERR_FILENO, "Internal error during type inference of\n");
```

Comment:
```suggestion
        jl_printf((JL_STREAM*)STDERR_FILENO, "Internal error: during type inference of\n");
```
I do want the new line before the type signature as these are long and i want to easy copy paste them
and to make the rest flow into either of the other two messages